content/en/account_management/billing/apm_tracing_profiler.md

```diff
@@ -49,19 +51,30 @@ Using 5 hosts and sending 30 million Indexed Spans, with 900 GB of total Ingeste
+### APM Pro Hosts, indexed spans, and extra ingested spans 
+
+Using 5 APM Pro hosts and sending 30 million Indexed Spans, with 900 GB of total Ingested Spans.
+
+| Billable Unit  | Quantity   | Price                                                                                           | Formula       | Subtotal              |
+|----------------|------------|-------------------------------------------------------------------------------------------------|---------------|-----------------------|
+| APM Pro Hosts      | 5          | $35 per host                                                                                    | 5 * $35       | $175                  |
+| Indexed Spans | 30 million | 5 million included with 5 APM hosts. $1.70 per million for additional 25 million Indexed Spans | 25 * $1.70    | $42.50                |
+| Ingested Spans | 900 GB          | 750 GB included with 5 APM hosts. $.10 per GB for additional 150 GB of Ingested Spans.                                                                                 | 150 * $.10      | $15                  |
+| Total          |            |                                                                                                 | $175 + $42.50 + $15 | **$232.50 per month** |
+
+### APM Enterprise Hosts with six profiled containers per host
 
-APM & Continuous Profiler for five hosts with six apps running in separate containers per each host.
+Using 5 APM Enterprise hosts, with the five hosts with six apps running in separate containers per each host.
 
 | Billable Unit  | Quantity   | Price                                                                                           | Formula       | Subtotal              |
 |----------------|------------|-------------------------------------------------------------------------------------------------|---------------|-----------------------|
-| APM & Continuous Profiler       | 5          | $40 per host                                                                                    | 5 * $40       | $200                  |
+| APM Enterprise Hosts       | 5          | $40 per host                                                                                    | 5 * $40       | $200                  |
 | Profiled containers  | 6 per host | $2 per additional container per host. In this case there are 6 - 4 = 2 additional containers for each host        | 2  * $2 * 5 hosts         | $20                   |
 | Total          |            |                                                                                                 | $200 + $20      | **$220 per month**    |
 
-### APM Hosts, Fargate, and indexed spans without profiler
+### APM Hosts, Fargate, and indexed spans
 
-Using 5 hosts, sending 20 million Indexed Spans, and have deployed APM on average 20 Fargate Tasks over the month.
+Using 5 APM hosts, sending 20 million Indexed Spans, and have deployed APM on average 20 Fargate Tasks over the month.
```

Comment:
```suggestion
Using five APM hosts, sending 20 million Indexed Spans, and have deployed APM on average 20 Fargate Tasks over the month.
```

Reply:
Not sure if it's just me, but I found this sentence slightly confusing. As an optional suggestion, `and have deployed APM on an average of 20 Fargate Tasks over the month` might be a little easier to understand if that still retains the same intended meaning.